docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  db:
```

Comment:
Not quite sure why mysql is needed.

Reply:
Ah, no it seems it's not needed. We could keep it to use in future versions, but I think it's nicer to just delete it. It's removed in f7ac8b2. If you disagree, let me know and I'll readd it.

Reply:
Mysql is so far not used in web-topcons. I'm OK with either keep it or not.